client/src/components/utils/LoginForm/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
```

Comment:
i think you don't need to extract login form  from login component. 
because this is not reusable component, 
i mean you will not reuse it 

please remove it and move it to login component